apps/persons/templates/persons/plugins/person.html

```diff
@@ -0,0 +1,18 @@
+{% load thumbnail i18n %}
+
+<div class="person-plugin">
+  <a class="person-plugin__avatar" href="{{ page.get_absolute_url }}"
+     title="{{ page.get_title }} {% trans "avatar" %}">
+    {% with related_plugins.portrait.0 as picture_plugin %}
+    {% thumbnail picture_plugin.picture 200x200 crop=picture_plugin.use_crop upscale=picture_plugin.use_upscale as thumbnail %}
+      <img src="{{ thumbnail.url }}" {{ picture_plugin.attributes_str }}
+           alt="{{ page.get_title }} {% trans "avatar" %}">
+    {% endwith %}
+  </a>
+  <div class="person-plugin__body">
+    <a href="{{ page.get_absolute_url }}" title="{{ page.get_title }}">
+      <h4 class="person-plugin__body__title">{{ page.person.get_full_name }}</h4>
```

Comment:
This should be SEO nicer but should be semantically not right if template base does not open an <h3> before, do we stand on idea than user may have to be carefull about HTML semantic changes he do ?

Reply:
Good catch. How can we do it?

Reply:
There is no real solution except adding this level information as a plugin attribute than user could change from frontend edition.

I saw your solution about using `{% with ... %}` in upstream template that include plugin to define a title level variable but finally it ends with user needing to override this template. It almost the same than just letting user to override templates to accommodate them to its title tree. So i think we should stand on this way to keep it simple